doc/releases/release-notes-3.7.rst

```diff
@@ -519,21 +519,10 @@ Drivers and Sensors
 
```

Comment:
As you need to do a new push anyhow, could you also fix this:
https://github.com/zephyrproject-rtos/zephyr/pull/75103#discussion_r1657410483